go/consensus/api/api.go

```diff
@@ -100,6 +103,19 @@ type Block struct {
 	Meta cbor.RawMessage `json:"meta"`
 }
 
+// Status is the current status overview.
+type Status struct {
+	NodeVersion string   `json:"node_version"`
```

Comment:
What has node version to do with a consensus backend? Maybe there should be a separate `GetStatus` API on the node's control service (see `go/control/api`) which returns something like this (including the `GetStatus` response for the consensus backend among other things):
```golang
type Status struct {
    SoftwareVersion string `json:"software_version"`
    // ... maybe some other versions

    Consensus consensus.Status `json:"consensus"`
    // ...
}
```
Consensus could then expose fields related to it like the consensus protocol version.

Reply:
Good idea, I'll add that.